server/index.js

```diff
@@ -135,14 +129,121 @@ var infer = function(req, res) {
+                channel,
+                channel,
+                channel
+            ], 2).squeeze();
+            roboflow.tf.dispose(channel);
+        }
+
+        var configuration = {
+            max_objects: Number.MAX_SAFE_INTEGER,
+            overlap: 0.3,
+            nms_threshold: 0.3,
+            threshold: 0.4,
+            tile: false,
+            tile_rows: 1,
+            tile_cols: 1,
+        };
+
+        if(req.query.overlap) {
+            req.query.overlap = parseFloat(req.query.overlap);
+            if(req.query.overlap > 1) req.query.overlap /= 100;
+            configuration.overlap = req.query.overlap;
+            configuration.nms_threshold = req.query.overlap;
+        }
+
+        if(req.query.confidence) {
+            req.query.confidence = parseFloat(req.query.confidence);
+            if(req.query.confidence > 1) req.query.confidence /= 100;
+            configuration.threshold = req.query.confidence;
+        }
+
+        //takes a tile query parameter like so tile=3x5
+        if(req.query.tile) {
+            if(false){
+                return res.json({
+                    error: "The inference server does not support this query parameter without a hosted image."
+                });
+            }
+            configuration.tile = true;
+
+            //ensure the tile query param looks like tile=300
+            if (!req.query.tile.match(/([0-9]+)/)){
+                return res.json({
+                    error: "Tile query parameter improperly formatted."
+                });
+            }
+            configuration.tile_rows = req.query.tile;
+            configuration.tile_cols = req.query.tile;
+        }
+
+
+        req.model.configure(configuration);
+
+        if(configuration.tile === true){
+            const tileDimensions = [configuration.tile_rows, configuration.tile_cols];
+            const paddedImage = padImage(tensor, tileDimensions);
+            const tiles = splitImage(paddedImage, tileDimensions);
+            var combinedResult = [];
+            async.eachOf(roboflow.tf.unstack(tiles), function(tile, index, cb) {
+                detect(req, res, start, tile, function(error, result) {
+                    if(!error){
+                        combinedResult.push(...result.predictions);
+                    } else {
+                        console.log("error after prediction ", error)
+                    }
+                    cb(null);
+                }, {
+                    index: index,
+                    tileDimensions: tileDimensions,
+                    imageDimensions: paddedImage.shape
+                });
+            }, function() {
+                res.json({
+                    predictions: combinedResult
+                });
+            });
+        } else {
+            detect(req, res, start, tensor, function(error, result){
+                if(error){
+                    console.log("error on detect", error)
```

Comment:
Was this error previously flowed through to the response?

Reply:
Yes it was, fixed in the latest version